EmployeesSalaries/EmployeesSalaries/Services/EmployeeService.cs

```diff
@@ -20,5 +26,18 @@ public List<IEmployee> GetEmployees()
         {
             return employees;
         }
+        public IReportsTo GetEmployeeReportsTo(int employeeId)
+        {
+            var emp = employeesReportsTo.FirstOrDefault(emp => emp.ReportsTo(employeeId));
+            if(emp == null)
+            {
+                throw new Exception("Not Assignable");
```

Comment:
no need to throw a new exception, just use a first instead first or default, and it will handle it for you. also no need for if statement